core/pom.xml

```diff
@@ -98,6 +98,10 @@
       <groupId>io.vavr</groupId>
       <artifactId>vavr</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.github.cdimascio</groupId>
+      <artifactId>dotenv-java</artifactId>
```

Comment:
(Major/Discussion)

Let's try to implement the feature request without additional dependency. Or did you already try and encounter impediments?

Reply:
https://github.com/cdimascio/dotenv-java
This is a no-dependency, pure Java project.
It only load environment variables from a .env file.

Reply:
This additional dependency is not yet convincing me.

Reply:
Hm okay, the code for regex actually is a lot.
https://github.com/cdimascio/dotenv-java/blob/master/src/main/java/io/github/cdimascio/dotenv/internal/DotenvParser.java

Also the classpath stuff looks like nothing we want to touch ourselves.
https://github.com/cdimascio/dotenv-java/blob/master/src/main/java/io/github/cdimascio/dotenv/internal/ClasspathHelper.java


I'm okay with an additional dependency.